runbotics/runbotics-actions-windows/src/automation/office/excel.action-handler.ts

```diff
@@ -239,6 +240,23 @@ export default class ExcelActionHandler extends StatefulActionHandler {
             .Insert();
     }
 
+    async deleteRows(input: ExcelDeleteRowsActionInput): Promise<void> {
+        const targetWorksheet = this.session.Worksheets(input?.worksheet ?? this.session.ActiveSheet.Name);
+
+        if (input.rowRange.match(/\d+:\d+/g)) return targetWorksheet.Rows(input.rowRange).Delete();
```

Comment:
you don't have to do this. check out delete columns - this line covers both: "F" and "C:K":

if (!Array.isArray(input.columnRange)) targetWorksheet.Columns(input.columnRange).Delete();

Reply:
so remove this line

Reply:
I was going to bring it up.  Array.isArray does not work no stings:
<img width="144" alt="image" src="https://github.com/runbotics/runbotics/assets/44839807/bcccd322-f35e-4c46-a304-6e1a7badfba9">
And action input return string:
<img width="96" alt="image" src="https://github.com/runbotics/runbotics/assets/44839807/e65c1050-9fae-4085-a638-0e42e78469e7">